plugins/modules/vmware_cluster_info.py

```diff
@@ -121,6 +121,7 @@
     type: dict
     sample: {
         "DC0_C0": {
+            "datacenter": "DC0",
```

Comment:
just a small request - Could you please mention `datacenter` info is added in 1.6.0 version?

Reply:
Thank you for your comment.
I added the info in the description in the RETURN block.  
How about this?